Resources/Private/Templates/Page/Chapter.html

```diff
@@ -26,9 +26,7 @@
 					</header>
 				</neos:contentElement.wrap>
 
-				<f:if condition="{node.properties.chapterImage}">
-					<media:image asset="{node.properties.chapterImage}" class="pull-right" alt="{node.properties.chapterImage.alternativeText}" title="{node.properties.chapterImage.title}" />
-				</f:if>
+				<media:image image="{node.properties.chapterImage}" class="pull-right" alt="{node.properties.chapterImage.alternativeText}" title="{node.properties.chapterImage.title}" />
```

Comment:
We don't need the condition anymore, since the Image ViewHelpers now behaves correctly and gracefully fals back to empty string, if no image is present:
https://github.com/neos/neos-development-collection/pull/1203